### PR TITLE
Psp 1843 side tray for side bar. 

### PR DESCRIPTION
**Pull request by devinleighsmith**
(NOTE: this review will remain in DRAFT until psp-1840 has been merged and this has been rebased).
Add side tray when side bar icons are clicked.
Add placeholder icons based on uxpin.
Add home icon with click functionality.
Add admin tray page with links to admin pages.
Add lease & license tray page with no functional links.
Simplify map html using grid.
Reorganize side bar components.

### CODE REVIEW

File: frontend/src/components/common/Label.tsx [hunk 1]
```diff
@@ -13,9 +12,17 @@ export const Label: React.FunctionComponent<ILabelProps & React.HTMLAttributes<H
...
-    <p {...rest} className={classNames('label', rest.className)}>
+    <StyledLabel {...rest} className={classNames('label', rest.className)}>
       {required && <span className="req">*</span>}
       {rest.children}
-    </p>
+    </StyledLabel>
   );
 };
+
+const StyledLabel = styled.p`
```
Comment (Fosol) [inline frontend/src/components/common/Label.tsx:22]: Wonder if we should start separating these into their own files as discussed last week.

File: frontend/src/components/common/LabelledIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+export const LabelledIcon = ({ icon, labelText }: ILabelledIconProps) => {
+  return (
+    <StyledLabelledIcon>
+      {icon}
+      <StyledLabel>{labelText}</StyledLabel>
+    </StyledLabelledIcon>
+  );
+};
+
+const StyledLabelledIcon = styled.div`
```
Comment (Fosol) [inline frontend/src/components/common/LabelledIcon.tsx:21]: Wonder if we should start separating these into their own files as discussed last week.

File: frontend/src/components/layout/SideNavBar/NavIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+const StyledLink = styled(Nav.Link)`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  svg {
+    min-width: max-content;
+  }
+`;
+
+const StyledLabel = styled.label`
```
Comment (Fosol) [inline frontend/src/components/layout/SideNavBar/NavIcon.tsx:50]: Wonder if we should start separating these into their own files as discussed last week.

**Review by Fosol — APPROVED** [review]

File: frontend/src/components/common/Label.tsx (hunk 1 above)
Comment (devinleighsmith) [reply frontend/src/components/common/Label.tsx]: Others are separated into their own files, but as discussed, doing that makes the most sense for large files. For small files moving styles to an external file increases the difficulty of parsing a component.

File: frontend/src/components/common/LabelledIcon.tsx (hunk 2 above)
Comment (devinleighsmith) [reply frontend/src/components/common/LabelledIcon.tsx:21]: see above.

File: frontend/src/components/layout/SideNavBar/NavIcon.tsx (hunk 3 above)
Comment (devinleighsmith) [reply frontend/src/components/layout/SideNavBar/NavIcon.tsx:50]: you wonder this a lot apparently.

**Pull request merged**